app/models/payroll_run.rb

```diff
@@ -16,7 +16,7 @@ def self.create_with_claims!(claims, attrs = {})
     ActiveRecord::Base.transaction do
       PayrollRun.create!(attrs).tap do |payroll_run|
         claims.each do |claim|
-          Payment.create!(payroll_run: payroll_run, claim: claim, award_amount: claim.award_amount)
+          Payment.create!(payroll_run: payroll_run, claims: [claim], award_amount: claim.award_amount)
```

Comment:
Would using a `has_one` association on `Payment` mean this change (and a bunch of similar changes in the specs) could come later when mulit-claim support is actually implemented?

Reply:
Yep. Changed it.